controller/src/main/java/io/pravega/controller/store/stream/ZKStreamMetadataStore.java

```diff
@@ -61,6 +67,19 @@
      * Since we use a 96 bit number for our counter, so
      */
     static final int COUNTER_RANGE = 10000;
+    static final String COUNTER_PATH = "/counter";
```

Comment:
I would prefer it if this new feature did not tie us even tighter to the ZK api. Ideally we should have some higher level abstraction that handles most of the logic and just have a ZK components for the specific list/path calls.